server/server.js

```diff
@@ -29,5 +31,44 @@ app.use(`/api/${process.env.SERVER_VERSION}/profile`, profile);
 
 // Run Server
 const host = process.env.HOST || `127.0.0.1`;
-const port = process.env.NODE_PORT || 3001;
+const port = process.env.NODE_PORT || 3009;
```

Comment:
Lets try to keep NODE_PORT at 3001 in the .env file and in the server. Try changing the port Socket.io listens on to 3002